api/server.spec.js

```diff
@@ -1,40 +1,57 @@
 const mongoose = require('mongoose');
+const request = require( 'supertest' );
+//const server = require( './api/server' );
+const Game = require( '../games/Game' );
 
-const Game = require('./games/Game');
-
-describe('The API Server', () => {
+describe('Game', () => {
   beforeAll(() => {
     return mongoose
       .connect('mongodb://localhost/test')
       .then(() => console.log('\n=== connected to TEST DB ==='))
       .catch(err => {
         console.log('error connecting to TEST database, is MongoDB running?');
       });
-  });
+  } );
+  
+  
 
   afterAll(() => {
     return mongoose
       .disconnect()
       .then(() => console.log('\n=== disconnected from TEST DB ==='));
   });
 
-  let gameId;
+  let gameId, game;
   // // hint - these wont be constants because you'll need to override them.
 
-  beforeEach(() => {
+  beforeEach(async() => {
```

Comment:
good way to keep game and game id and since there is 2 of them using async helped instead of a direct return!